src/libsystemd/sd-journal/catalog.c

```diff
@@ -618,8 +618,7 @@ int catalog_get(const char* database, sd_id128_t id, char **_text) {
         r = 0;
 
 finish:
-        if (p)
-                munmap(p, st.st_size);
+        munmap(p, st.st_size);
```

Comment:
Please void-ify the statement, since you're already touching that code, i.e.

```suggestion
        (void) munmap(p, st.st_size);
```